tools/dataform_assertions/definitions/unit_test/date_format_row_assertions_unit_test.sqlx

```diff
@@ -0,0 +1,38 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+config {
+    type: "view",
+    schema: "dev_data_zone",
+    name: "date_format_row_assertions_unit_test",
+    tags:["date_format_row_assertions_unit_test","all_test"],
+    assertions: {
+        nonNull: ["input"],
+        rowConditions: 
+        [
+            date_assertions.test_date_format("input", "yyyy/mm/dd")
+        ]
+    }
+}
+
+/* 
+    This assertion should fail 1 assertion
```

Comment:
Remove comment

Reply:
remove similar comments from other files as well since our expected_output column holds this info now